monkey_island/cc/services/config.py

```diff
@@ -794,29 +797,42 @@
     }
 }
 
+ENCRYPTED_CONFIG_ARRAYS = \
+    [
+        ['basic', 'credentials', 'exploit_password_list'],
+        ['internal', 'exploits', 'exploit_lm_hash_list'],
+        ['internal', 'exploits', 'exploit_ntlm_hash_list']
+    ]
+
 
 class ConfigService:
+    default_config = None
+
     def __init__(self):
         pass
 
     @staticmethod
-    def get_config(is_initial_config=False):
+    def get_config(is_initial_config=False, should_decrypt=True):
```

Comment:
Please document when `get_config` and `get_config_value` should each be used

Reply:
fixed